py/aou_workbench_client/concepts.py

```diff
@@ -62,6 +64,129 @@
     ('Vocabulary', 'vocabulary_id'),
     ('Count', 'count_value')]
 
+_CONCEPT_TABLE_HTML_TEMPLATE = """
+<script language="javascript">
+  var selected_row_id = null;
+  var selected_data = null;
+  
+  var old_selected_color = null;
+  
+  function select_concept(id, name, domain, vocabulary, standard) {
+    max_results = document.getElementById('max_results');
```

Comment:
Unless ipynb sandboxes your JS somehow, you may want some mechanism of scoping these element lookups. For example, try rendering two of these widgets in the same notebook and test what happens - if things aren't sandboxed, you're likely to see problems here.

If Ipython doesn't prescribe a way of fixing this, then one possible solution is to wrap everything in a container with a class containing a random nonce, then including that in your element lookups.

Reply:
Added a TODO for this, JIRA issue RW-796